frameit/lib/frameit/config_parser.rb

```diff
@@ -80,11 +80,11 @@ def validate_values(values)
           when 'color'
             UI.user_error!("Invalid color '#{value}'. Must be valid Hex #123123") unless value.include?("#")
           when 'padding'
-            unless value.kind_of?(Integer) || value.split('x').length == 2
-              UI.user_error!("padding must be type integer or pair of integers of format 'AxB'")
+            unless value.kind_of?(Integer) || value.split('x').length == 2 || (value.end_with?('%') && value.to_f > 0)
+              UI.user_error!("padding must be type integer or pair of integers of format 'AxB' or a percentage of screen size")
             end
           when 'font_scale_factor'
-            UI.user_error! "font_scale_factor must be numeric" unless value.kind_of?(Numeric)
+            UI.user_error!("font_scale_factor must be numeric" unless value.kind_of?(Numeric))
```

Comment:
I think this closing parenthesis is in the wrong place?  Should be after closing quote, and before the `unless`. This seems to be why CI is complaining.